src/App.ts

```diff
@@ -979,10 +943,10 @@ function defaultErrorHandler(logger: Logger): ErrorHandler {
   };
 }
 
-function singleTeamAuthorization(
+function singleAuthorization(
```

Comment:
The rename is because this function addresses authorization on a single team OR a single enterprise. I considered having two separate functions for those purposes, but realized the body would look mostly identical except for the value of `isEnterpriseInstall` in the return value. That could easily be consolidated because `isEnterpriseInstall` is actually passed to the `Authorize` function as a property of `source` anyway.